### PR TITLE
Styling for file export page

### PR DESCRIPTION
**Pull request by axemonkey**
screen shot 2016-09-01 at 17 04 58

### CODE REVIEW

File: dashboard/templates/admin/prototype/export.html [hunk 1]
```diff
@@ -1,13 +1,136 @@
...
+          </label>
+          <input autocomplete="off" class="form-control" id="id_date_month" maxlength="2" name="date_month" placeholder="MM" size="3" type="text">
+          <input autocomplete="off" class="form-control" id="id_date_year" maxlength="4" name="date_year" placeholder="YYYY" size="5" type="text">
+        </div>
+
+        <div class="form-group">
+          <label class="form-label" for="id_project">Project:</label>
+          <select class="form-control" id="id_project" name="project">
+            <option value="" selected="selected">---------</option>
+            <option value="1">Admin / team lead</option>
```
Comment (cliffxuan) [inline dashboard/templates/admin/prototype/export.html]: this needs to be generated from a loop through all the `Project` objects.

Comment (axemonkey) [reply dashboard/templates/admin/prototype/export.html]: Oh yes, I remember now.

Comment (cliffxuan) [reply dashboard/templates/admin/prototype/export.html]: i can send you a pull request if you like.

Comment (axemonkey) [reply dashboard/templates/admin/prototype/export.html]: It's OK, I had most of it and then Josh helped me with which object I needed to loop through. 👍

File: dashboard/templates/admin/prototype/export.html [hunk 2]
```diff
@@ -1,13 +1,65 @@
...
+          </label>
+          <input autocomplete="off" class="form-control" id="id_date_month" maxlength="2" name="date_month" placeholder="MM" size="3" type="text">
+          <input autocomplete="off" class="form-control" id="id_date_year" maxlength="4" name="date_year" placeholder="YYYY" size="5" type="text">
+        </div>
+
+        <div class="form-group">
+          <label class="form-label" for="id_project">Project:</label>
+          <select class="form-control" id="id_project" name="project">
+            <option value="" selected="selected">---------</option>
+            {% for value, display in form.fields.project.choices %}
```
Comment (cliffxuan) [inline dashboard/templates/admin/prototype/export.html:31]: nice change. still a little problem. `---------` is appearing twice. also projects might be sorted by name?

Comment (axemonkey) [reply dashboard/templates/admin/prototype/export.html:31]: Double `---------` fixed. Not sure how to fix the sorting...

Comment (cliffxuan) [reply dashboard/templates/admin/prototype/export.html:31]: `forms.py`

```
class ExportForm(forms.Form):
    date = MonthYearField(required=True)
    project = forms.ModelChoiceField(
        queryset=Project.objects.visible().order_by('name'),
        required=True)
```

**Comment by cliffxuan** [comment]
👍

**Pull request merged**